Scripts/ExtractDomainFromUrlFormat/ExtractDomainFromUrlFormat.yml

```diff
@@ -1,21 +1,32 @@
+args:
+- default: false
+  description: The URL(s) or Email(s) to process
+  isArray: true
+  name: input
+  required: false
+  secret: false
+- default: false
```

Comment:
add options (true, false)

Reply:
Do you mean to write the options in the argument description?

Reply:
no, i mean as `predefined`

Reply:
Done